inst/WORDLIST

```diff
@@ -0,0 +1,122 @@
+ALLUPPERCASE
+Addins
+Backport
+CMD
+Checkstyle
+Config
+Ctrl
+Cyclomatic
+Deprecations
+EQ
+ESS
+Github
+Linter
+Linters
+MegaLinter
+ORCID
+RMarkdown
+RStudio
+Rhtml
+Rmarkdown
+Rmd
+Rnw
+Rproj
+Rrst
+Rstudio
+Rtex
+Rtxt
+SARIF
+SublimeLinter
+Tidyverse
+UpperCamelCase
+Wickham
+XPath
+XPaths
+YAML
+Yihui
+addin
+addins
+alllowercase
+backports
+backticked
+behaviour
+bookdown
+bugfix
+checkstyle
+codecov
+codinghorror
+coercions
+config
+covr
+customizable
+customizations
+cyclomatic
+dcf
+de
+debian
+dplyr
+dragosmg
+eg
+envvar
+eval
+favour
+flycheck
+github
+https
+igraph
+importFrom
+infixes
+io
+jwz
+knitr
+labelled
+lang
+languageserver
+linter
+linters
+lintr's
+localisation
+lowerCamelCalse
+magrittr
+michaelchirico
+mis
+nd
+nodeset
+nolint
+parameterised
+patilindrajeets
+pre
+programmatically
+qmd
+readably
+regexes
+repo
+rlang
+roxygen
+sandboxing
+spilt
+src
+stopifnot
+styler
+subdir
+syntastic
```

Comment:
is syntastic a word?

Reply:
It's name of a Vim plugin: https://github.com/vim-syntastic/syntastic

Reply:
does `spelling` support comments in the `WORDLIST`? it could be good to annotate cases like this so we don't forget

Reply:
IINM, it's a debian/control file, so, I don't think so.